internal/config/load.go

```diff
@@ -96,7 +111,7 @@ func mergeAll(fs afero.Fs, repo *git.Repository) (*viper.Viper, error) {
 		return nil, err
 	}
 
-	if err := merge("lefthook-local", "", extends); err == nil {
+	if err := mergeOne([]string{".lefthook-local", "lefthook-local"}, "", extends); err == nil {
```

Comment:
Let's do the same here. 

```suggestion
	if err := mergeOne([]string{"lefthook-local", ".lefthook-local"}, "", extends); err == nil {
```